outport/process/transactionsfee/transactionsFeeProcessor.go

```diff
@@ -258,6 +258,22 @@ func (tep *transactionsFeeProcessor) prepareRelayedTxV3WithResults(txHashHex str
 		refundsValue.Add(refundsValue, scr.Value)
 	}
 
+	isRelayedV3 := len(txWithResults.GetTxHandler().GetUserTransactions()) > 0
```

Comment:
should also check if it has refunds here?

Reply:
In this case no, the refunds are in the Smart Contract Results map.